plugins/kotlin/idea/tests/testData/refactoring/introduceConstant/binaryExpression/stringAddition.kt.after

```diff
@@ -0,0 +1,9 @@
+// EXTRACTION_TARGET: property with initializer
+val a = 1
+
+private const val s = "1" + "1"
+
+fun foo(): String {
+    val x = 1+1
```

Comment:
Does this test actually verify that the unrelated code is reformatted? Or only the actual programmatic elements become diffed?

Reply:
This comes from me orienting myself on the other tests, see `extractTrue` or `extractIntegerLiteral` which I copied from introduce property